balboa-http/src/main/scala/com/socrata/balboa/server/ResponseWithType.scala

```diff
@@ -9,15 +9,21 @@ object ResponseWithType {
   val json = "application/json; charset=utf-8"
   val protobuf = "application/x-protobuf"
 
+  val BadRequestCode = 400
+  val InternalErrorCode = 500
+  val NotFoundCode = 404
+  val NotAcceptableCode = 406
```

Comment:
Why not use:

    org.apache.http.HttpStatus.SC_BAD_REQUEST
    org.apache.http.HttpStatus.SC_INTERNAL_SERVER_ERROR
    org.apache.http.HttpStatus.SC_NOT_FOUND
    org.apache.http.HttpStatus.SC_NOT_ACCEPTABLE

Or org.eclipse.jetty.http.HttpStatus has them too.